app/controllers/users_controller.rb

```diff
@@ -4,24 +4,23 @@ def new
     end
```

Comment:
I wonder if something is happening to the indentation. I had to fix on my branch too.